web-ui/src/main/resources/catalog/components/thesaurus/ThesaurusService.js

```diff
@@ -91,12 +123,36 @@
               );
             };
 
+            var getUILangs = function() {
+                  var currentUILang_3char = gnLangs.detectLang(
+                                              gnGlobalSettings.gnCfg.langDetector,
+                                              gnGlobalSettings
+                                            );
+                  var currentUILang2_3char = gnCurrentEdit.allLanguages.iso2code[currentUILang_3char].replace("#","");
```

Comment:
This looks to be failing when editing an english record using UI in french for example

![image](https://user-images.githubusercontent.com/1701393/89010514-04663a00-d30f-11ea-8671-1f3331d4ba0c.png)

Something to fix?